lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/project/MavenProjectCache.java

```diff
@@ -405,8 +406,8 @@ public MavenProject getSnapshotProject(DOMDocument document, String profileId, b
 		 * 
 		 * @return A ProjectBuildingRequest object
 		 */
-		public ProjectBuildingRequest newProjectBuildingRequest(File projectFile) {
-			return newProjectBuildingRequest(true, projectFile);
+		public ProjectBuildingRequest newProjectBuildingRequest(File projectFile, Collection<ModelProblem> problems) {
```

Comment:
Is the `problems` a "collector" here that will be populated by execution of this method? If so, can you please rename it to `problemsCollector` ?

Reply:
Done.